docs/sigs/Certification.md

```diff
@@ -15,36 +15,49 @@ We use [mattermost SIG/Certification](https://chat.almalinux.org/almalinux/chann
 
 **Where and when we meet**
 
-Meetings are approximately every fortnight. If there's nothing confidential in the next meeting agenda a SIG member will happily welcome you along and provide time and meeting URL information.
+Meetings are approximately every fortnight. If there's nothing confidential in the next meeting agenda a SIG member will happily welcome you along and provide time and meeting URL information. You can also find details about the next meeting on the events site: [Certification SIG meetings](https://events.almalinux.org/category/6/)
 
 ## Activities, projects, and deliverables
 
-Current deliverables are:
-* [AlmaLinux OS Foundation Hardware Certification Program](https://docs.google.com/document/d/1r33jRSavBa9yiI8FlRDHbpOhhW83xMFrsR_bqTnSJPI/edit?usp=sharing) - the document describing the process for certifying hardware.
-* The [AlmaLinux Hardware Certification Suite](https://github.com/AlmaLinux/Hardware-Certification-Suite), where the Ansible rules for exercising a test suite is maintained.
-
-Once active, the SIG will be taking test results and issuing certifications.
+The SIG is responsible for a few different things, and maintains a few repos:
+
+-   [Hardware Certification Program](https://almalinux.org/certification/hardware-certification/hardware-certification-program/)
+    -   This program allows for the Certification SIG to engage with IHVs to certify hardware for AlmaLinux OS.
+-   [AlmaLinux/Certifications](https://github.com/AlmaLinux/certifications)
+    -   This repo is the primary point of contact for the SIG, acts as the holder of certification results, and is the location for certification requests that are not initiated by the foundation or the IHV. It is also the source for the Hardware Certification Program documentation.
```

Comment:
I don't like "holder of", sounds like a person ;-)

How about "holding place for",  "storage area for", or re-use "location of"....?

Reply:
that's a good call out. Adjusted!